src/pages/installing/license-tracking.md

```diff
@@ -8,7 +8,7 @@ toc: true
 
 License Service is required for monitoring and measuring license usage of ACD in accord with the pricing rule for IBM containerized software.
 License usage is based on a Virtual Processing Core (VPC) metric. For ACD, it is the top-level pod `ibm-wh-acd-acd` that will be metered for VPC usage.
-The usage will be the capacity of the node where this pod runs. For example, ACD running in a cluster with 16 CPU/node has license usage = 16 VPC. The ACD CPU limit can be adjusted with the ```spec.resources.limits.cpu```. For more information on configuration see [Configuration](/management/configuring/).
+The usage will be the capacity of the node where this pod runs. For example, ACD running in a cluster with 16 CPU/node has license usage = 16 VPC. The ACD CPU limit can be adjusted by the number of replicas configured and the size of the nodes where ACD runs. For more information on configuration see [Configuration](/management/configuring/).
```

Comment:
Looks good to me, but this tracking will only work with the old operator pods, not the merative operator pods. The top level pod for the merative operator will start with `merative-acd-acd` so the tracking won't find any product charged containers (see line 10 above).

Reply:
metering should still work (annotation will have to move to that new pod name) and then  doc will just need to be changed w/ merative release to indicate that. But this change is just to fix the fact that we don't allow overrides on spec.resources.limits.cpu.